packages/fluentui/docs/src/components/ComponentDoc/ComponentDoc.tsx

```diff
@@ -152,6 +152,13 @@ class ComponentDoc extends React.Component<ComponentDocProps, ComponentDocState>
             activeIndex={currentTabIndex}
             items={tabs}
             style={{ marginTop: '0.5rem', background: 'none', border: 'none' }}
+            variables={{
+              underlinedColorHover: teamsTheme.siteVariables.colors.black,
+              color: teamsTheme.siteVariables.colors.grey[500],
+              colorActive: teamsTheme.siteVariables.colors.black,
+              activeUnderlinedBorderBottomColor: teamsTheme.siteVariables.colors.black,
+              underlinedWrapperColorHover: teamsTheme.siteVariables.colors.black,
```

Comment:
Surprisingly this has no effect but it has to
`underlinedWrapperColorHover: teamsTheme.siteVariables.colors.black,`